mlf_core/config/config.py

```diff
@@ -44,8 +45,9 @@ def config_general_settings() -> None:
         already_configured = False
         settings = {}
         if ConfigCommand.check_mlf_core_config_dir_exists() and 'GITHUB_ACTIONS' not in os.environ:
-            already_configured = True
-            settings = load_yaml_file(ConfigCommand.CONF_FILE_PATH)
+            if os.path.exists(Path(ConfigCommand.CONF_FILE_PATH)):
```

Comment:
Could add this to the check before?
a and b and c

Reply:
Yeah thought about it and decided to put it here to reduce the length of the if statement and make it a bit more readable?

Reply:
I'd suggest to use line breaks and put it into one big IF